api/scenario_api/launch/scenario.launch.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<launch>
+  <!-- Essential parameters -->
+  <arg name="map_path" description="point cloud and lanelet2 map directory path"/>
+  <arg name="vehicle_model" default="jpntaxi" />
+  <arg name="sensor_model" default="aip_xx1" />
+  <arg name ="vehicle_id" value="$(optenv VEHICLE_ID default)" />
+  <!-- Optional parameters -->
+  <arg name="rviz" default="true" description="launch rviz"/>
+  <arg name="initial_engage_state" default="false" />
+  <arg name="use_object_recognition" default="true" />
+
+  <!-- Vehicle Description -->
+  <group>
+    <include file="$(find-pkg-share vehicle_launch)/launch/vehicle_description.launch">
```

Comment:
Ah, that's interesting, have to look into this deeper. Potentially a better alternative to hardcoding the Lexus.

Reply:
good point, yes the indirection happens in that file
```xml
<launch>
  <arg name="sensor_model"/>
  <arg name="vehicle_model"/>
...
  <group ns="/vehicle_info">
  <rosparam file="$(eval find(arg('vehicle_model')+'_description')+'/config/vehicle_info.yaml')" command="load"/>
```

it's already ported; cf https://github.com/tier4/autoware_launcher.iv.universe/pull/4